### PR TITLE
Add possibility to change drain command parameters

### PR DESCRIPTION
**Pull request by wanderaBrno**
For our purpose we need to also add parameter --delete-local-data, so we little bit refactored the script. With this change it's possible to change/alter parameters.

### CODE REVIEW

**Comment by wanderaBrno** [comment]
I've added default values to example daemonset env vars, but can add it directly to the code

**Comment by JoelSpeed** [comment]
I think it's good to have in the example and in the code, if you could please 😄

**Comment by wanderaBrno** [comment]
> Hey, thanks for opening this PR. I think this is a great idea, though I'd be tempted to set a sensible default if the environment variable is not set, WDYT? If we default to the existing config, we won't make a breaking change

@JoelSpeed reworked little bit

File: docker_entrypoint.py [hunk 1]
```diff
@@ -22,9 +24,9 @@ def main():
             "http://169.254.169.254/latest/meta-data/spot/termination-time"
         )
         if response.status_code == 200:
-            kube_command = ['kubectl', 'drain', node_name,
-                            '--grace-period=120', '--force',
-                            '--ignore-daemonsets']
+            kube_command = ['kubectl', 'drain', node_name]
+
```
Comment (JoelSpeed) [inline docker_entrypoint.py]: Nit: probably don't need this extra blank line

**Comment by wanderaBrno** [comment]
@JoelSpeed when you merge, can you please release new version? Thanks in advance!

**Comment by wanderaBrno** [comment]
Hello @JoelSpeed, is there anything else we can do for merging this PR?

**Comment by JoelSpeed** [comment]
One final thought, perhaps we could add a new note to the readme explaining the new functionality?

**Comment by wanderaBrno** [comment]
@JoelSpeed I've tried to add simple paragraph for this option.

**Comment by wanderaBrno** [comment]
@JoelSpeed forgot to push 2 days ago :( please check now

**Comment by jsedlace** [comment]
Hi @JoelSpeed, can you make a new release containing code from this PR, please?

**Comment by theobarberbany** [comment]
@jsedlace Here you go :) https://github.com/pusher/k8s-spot-termination-handler/releases/tag/v0.2.0

**Pull request merged**